scraper_data.py

```diff
@@ -21,8 +22,14 @@ def main(counties: Tuple[str,...], output:str) -> None:
         counties = COUNTY_NAMES
 
     # Run each scraper's get_county() method. Assign the output to out[county]
+    failed_counties = []
     for county in counties:
-        out[county] = data_scrapers.scrapers[county].get_county()
+        try:
+            out[county] = data_scrapers.scrapers[county].get_county()
+        except:
+            failed_counties.append(county)
+
+    if failed_counties: send_webhook(failed_counties)
```

Comment:
Quick ’n dirty thoughts as you are refining this tonight:

1. We should still log the error to stderr, either here or above in the `except` clause.

2. We should exit with a non-zero code to signify that things failed (e.g. `sys.exit(1)`). I think it might also be nice to have different exit codes for “all counties failed” (so we got nothing) and “some counties failed” (so we got something worth using, but there were still issues).